ci/azure-pipelines-docs-integration.yml

```diff
@@ -59,39 +61,6 @@ stages:
           - bash: ./ci/checks/check_for_docs_deps_changes.sh
             name: CheckDocsDependenciesChanges
 
-  - stage: custom_checks
-    dependsOn: scope_check
```

Comment:
These have been moved to `azure-pipelines-dev.yml`

Reply:
(I think you mean `azure-pipelines-docs.yml` right?)